spec/converters/shopify/variant_converter_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+module Shopify
+  RSpec.describe VariantConverter do
+    include_context 'spree_builders'
+
+    let(:spree_variant) { build_spree_variant }
+
+    describe '.initialize' do
+      subject { described_class.new(variant: spree_variant) }
+
+      it "successfully does it's thing" do
```

Comment:
Totally dig these as beginner tests to really flush out the object but not sure if they are a need after the first public method is under test. Moreover, what does 'successfully does it's thing' tell the reader? What do you think?